stack_definition.json

```diff
@@ -135,155 +139,243 @@
 			"inputs": [
 				{
 					"name": "resource_group_name",
-					"value": "ref:../1 - Account Infrastructure Base/outputs/security_resource_group_name"
+					"value": "ref:../../members/1%20-%20Account%20Infrastructure%20Base/outputs/security_resource_group_name"
 				},
 				{
 					"name": "use_existing_resource_group",
 					"value": true
 				},
-				{
-					"name": "ibmcloud_api_key",
-					"value": ""
-				},
 				{
 					"name": "region",
 					"value": "ref:../../inputs/region"
 				},
 				{
-					"name": "region",
-					"value": "ref:../../inputs/region"
+					"name": "prefix",
+					"value": "ref:../../inputs/prefix"
 				},
 				{
 					"name": "existing_kms_instance_crn",
 					"value": "ref:../../inputs/existing_kms_instance_crn"
 				}
 			],
 			"name": "2a - Security Service - Key Management",
-			"version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.dafa6c99-436a-4d01-a379-8ef818af59a1-global"
+			"version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.e7f105c4-8af4-4238-a98d-e89999ff14c8-global"
 		},
 		{
 			"inputs": [
+				{
+					"name": "resource_group_name",
+					"value": "ref:../../members/1%20-%20Account%20Infrastructure%20Base/outputs/security_resource_group_name"
+				},
+				{
+					"name": "existing_resource_group",
+					"value": true
+				},
+				{
+					"name": "cos_instance_name",
+					"value": "core-services"
+				},
 				{
 					"name": "prefix",
 					"value": "ref:../../inputs/prefix"
-				},
+				}
+			],
+			"name": "2b - Security Service - Object storage",
```

Comment:
hmm, should this have `Security Service` in the name? Its not really one of the security services (although it is a member of the core security services stack)

Reply:
This is the naming format from the rag stack...

Reply:
right, but not what we used in core security services - we should run by team (Allen) for final review on member names